Makefile

```diff
@@ -1,6 +1,6 @@
-.PHONY: build test clean prepare update
+.PHONY: build test clean docker
 
-GO=CGO_ENABLED=0 go
+GO=CGO_ENABLED=0 GO111MODULE=on go
```

Comment:
Just a minor comment on the GO111MODULE env var. That indicates to the compiler that we're building the module in the GOPATH. If we build the module outside of the GOPATH, we don't need to set this. Has that been tried? I'm not sure what will come of a variable named for Go 1.11 once Go 1.12 comes out. It could go away and we'd have to make changes whereas if we have a solution that doesn't rely on this, could be more robust. FWIW